lib/bike_brigade_web/live/campaign_live/index.ex

```diff
@@ -21,14 +26,33 @@ defmodule BikeBrigadeWeb.CampaignLive.Index do
      |> assign(:page, :campaigns)
      |> assign(:page_title, "Campaigns")
      |> assign(:current_week, current_week)
-     |> assign(:campaigns, fetch_campaigns(current_week))}
+     |> assign(:campaigns, fetch_campaigns(current_week))
+     |> assign(:campaign_task_counts, Delivery.get_total_tasks_and_open_tasks(current_week))
+    }
   end
 
   @impl true
   def handle_params(params, _url, socket) do
     {:noreply, apply_action(socket, socket.assigns.live_action, params)}
   end
 
+  @broadcasted_infos [
+    :task_created,
+    :task_deleted,
+    :task_updated,
+    :campaign_rider_created,
+    :campaign_rider_deleted
+  ]
+
+  @impl Phoenix.LiveView
+  def handle_info({event, entity}, socket) when event in @broadcasted_infos do
+    if entity_in_campaigns?(socket.assigns.campaigns, entity.campaign_id) do
+      {:noreply, refetch_and_assign_data(socket)}
+    else
+      {:noreply, socket}
+    end
+  end
+
```

Comment:
Thanks for adding this!

two feedbacks:

1) I think this should go after the handle_events
2) It needs a 
```
@impl Phoenix.LiveView
  @doc "silently ignore other kinds of messages"
  def handle_info(_, socket), do: {:noreply, socket}
```
after it to silently ignore messages we don't care about (or else the LV crashes :()